mlflow/server/handlers.py

```diff
@@ -790,6 +791,29 @@ def _log_param():
     return response
 
 
+@catch_mlflow_exception
+@_disable_if_artifacts_only
+def _log_inputs():
+    request_message = _get_request_message(
+        LogInputs(),
+        schema={
+            "run_id": [_assert_required, _assert_string],
+            "datasets": [_assert_required, _assert_array],
+        },
+    )
+    run_id = request_message.run_id or request_message.run_uuid
```

Comment:
Remove `run_uuid` since it doesn't exist in the proto: https://sourcegraph.com/github.com/mlflow/mlflow@8549564b7e474b8350dd82962d45ee5def92c156/-/blob/mlflow/protos/service.proto?L1105